.betterer.ts

```diff
@@ -86,6 +86,7 @@ function countEslintErrors() {
       '@emotion/syntax-preference': [2, 'object'],
       '@typescript-eslint/no-explicit-any': 'error',
       '@grafana/no-aria-label-selectors': 'error',
+      '@grafana/theme-token-usage': 'error',
```

Comment:
Would `error` prevent changes/increase in numbers?

Reply:
Yes, I have to think in another way of doing this 😅

Reply:
Maybe creating a new betterer suite, and just setting the constraints to `bigger` may work https://phenomnomnominal.github.io/betterer/docs/tests

That way you can also group the theme stuff in a single suite.

Reply:
think we should avoid this. this would block people deleting code for example. there's no constraint we want on this metric, we just want to track it. therefore we shouldn't use betterer 👍